### PR TITLE
REST Refactor

### PR DESCRIPTION
**Pull request by arfusop**
* Ripped out TypeScript (for now...)
* Remove GraphQL & Apollo, and any other related dependencies
* Refactored server from ^ to be a REST api

### CODE REVIEW

**Comment by arfusop** [comment]
[Refactor the Node server from graphql to rest](https://trello.com/c/S9TBbWlq/35-refactor-the-node-server-from-graphql-to-rest)

**Review by SamiFdev — APPROVED** [review]
You name all your variables etc so specifically. I need to do more of that. I tend to name it well but loosely and then it gets confusing down the line because I have something using 2-3 of the same words. A lot of good stuff to take away from this.

File: server/controllers/clients/updateClientMeetingEndpoint.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+import Clients from '../../models/Clients.js'
+import { phases } from './utils.js'
+import { handleInternalError } from '../../../utils/buildResponse.js'
+
+export default async (req, res) => {
+    try {
+        const { eventUri, inviteeUri } = req.body
+        const config = {
+            headers: {
```
Comment (KevinLaCarrubba) [inline server/controllers/clients/updateClientMeetingEndpoint.js:11]: just change it to authHeader

File: server/controllers/clients/utils.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const phases = {
+    PHASE_1: 'Phase 1: Information Gathering',
+    PHASE_2: 'Phase 2: Structure & Design',
+    PHASE_3: 'Phase 3: Initial Development',
+    PHASE_4: 'Phase 4: Testing/QA',
+    PHASE_5: 'Phase 5: Post Launch Maintenance',
+    PHASE_6: 'Phase 6: New Version Development',
+    PHASE_7: 'Phase 7: Terminated'
```
Comment (KevinLaCarrubba) [inline server/controllers/clients/utils.js:8]: Terminated or Completed ? Termination makes me feel like we deleted it

File: server/controllers/users/getAllUsersEndpoint.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export default async (req, res) => {
+    console.log('all users return here')
+    return res.status(200).json({ users: [1, 2] })
```
Comment (KevinLaCarrubba) [inline server/controllers/users/getAllUsersEndpoint.js:3]: what does this [1,2] return ?

File: utils/buildResponse.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+const HTTP_CODES = {
```
Comment (KevinLaCarrubba) [inline utils/buildResponse.js:1]: this is dope

File: server/controllers/clients/updateClientMeetingEndpoint.js (hunk 1 above)
Comment (arfusop) [reply server/controllers/clients/updateClientMeetingEndpoint.js:11]: dont even

File: server/controllers/clients/utils.js (hunk 2 above)
Comment (arfusop) [reply server/controllers/clients/utils.js:8]: I always saw Terminated as end of support, regardless of the reason. We can certainly rename the label to something everyone feels more comfortable having

File: server/controllers/users/getAllUsersEndpoint.js (hunk 3 above)
Comment (arfusop) [reply server/controllers/users/getAllUsersEndpoint.js:3]: it's dummy data. Returns a `user` object with an array of 1 and 2. This is a placeholder for now cuz i was just testing routes to make sure my node server was set up to run them

**Pull request merged**